packages/react/src/components/ToggleGroup/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import * as icons from '@navikt/aksel-icons';
+
+import { Button } from '../Button';
+
+import { ToggleGroup } from '.';
+
+const icon = (
+  <svg
+    viewBox='0 0 24 24'
+    fill='none'
+    xmlns='http://www.w3.org/2000/svg'
+  >
+    <path
+      fillRule='evenodd'
+      clipRule='evenodd'
+      d='M12 0c6.627 0 12 5.373 12 12s-5.373 12-12 12S0 18.627 0 12 5.373 0 12 0Zm0 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2Zm5.047 5.671 1.399 1.43-8.728 8.398L6 14.02l1.395-1.434 2.319 2.118 7.333-7.032Z'
+      fill='currentColor'
+    />
+  </svg>
+);
+
+const AkselIcon = icons.AirplaneFillIcon;
+const AkselIcon2 = icons.NewspaperFillIcon;
+const AkselIcon3 = icons.BrailleIcon;
+const AkselIcon4 = icons.BackpackFillIcon;
+
+export default {
+  title: 'Felles/ToggleGroup',
+  component: ToggleGroup,
+} as Meta;
+
+export const Preview: StoryFn<typeof ToggleGroup> = (args) => {
+  return (
+    <ToggleGroup
+      {...args}
+      defaultValue='Peanut'
+    >
+      <ToggleGroup.Item>Peanut</ToggleGroup.Item>
+      <ToggleGroup.Item>Walnut</ToggleGroup.Item>
+      <ToggleGroup.Item>Pistachio 🤤</ToggleGroup.Item>
+    </ToggleGroup>
+  );
+};
+
+Preview.args = {
+  size: 'medium',
+  defaultValue: 'test',
```

Comment:
Suggest removing this default value as it does not match the example used in preview.
```suggestion
  defaultValue: 'test',
```